python/ray/train/accelerators/torch.py

```diff
@@ -0,0 +1,173 @@
+import logging
+from typing import Any, Dict, Optional
+
+import torch
+from torch.nn.parallel import DistributedDataParallel
+from torch.utils.data import (
+    DistributedSampler,
+    DataLoader,
+    IterableDataset,
+    SequentialSampler,
+)
+
+from ray import train
+
+from .base import Accelerator
+
+logger = logging.getLogger(__name__)
+
+
+class TorchAccelerator(Accelerator):
```

Comment:
What do you think about moving all the torch related code to a single torch module? Then when importing this module, we can raise an error if torch is not installed directly in the `__init__.py`.

Reply:
Fixed